js/constants/deployment.js

```diff
@@ -10,6 +10,14 @@ export const STATE_ABORTING = 'Aborting';
 export const STATE_COMPLETED = 'Completed';
 export const STATE_FAILED = 'Failed';
 
+// These constants are a subset of the deploynaut states
+// this is because an approval can be bypassed and also
+// several deployment states maps to 'approved'.
+export const APPROVAL_SUBMITTED = 'submitted';
```

Comment:
ah wait, maybe not. Nevermind. I got confused because there is `APPROVAL_SUBMITTED` and `STATE_SUBMITTED` and such which are very similar ;-)